standalone/src/models/cart/cart/discounted-line-item-portion/fields-config.ts

```diff
@@ -10,15 +15,15 @@ export const restFieldsConfig: TModelFieldsConfig<TDiscountedLineItemPortionRest
   {
     fields: {
       discount: fake(() => ReferenceRest.presets.cartDiscountReference()),
-      discountedAmount: fake(() => Money.random()),
+      discountedAmount: fake(() => MoneyRest.random()),
```

Comment:
Not related to the `Price` migration but I noticed we also need to start using the new `Money` models so I took the opportunity to update some of the models I identified.